# the time stamp of getting this data in the ROS system
time stamp

# the time stamp from GPS/RTK
string UTCTimeData
